charts/bridgeai-prediction-service/Chart.yaml

```
@@ -19,14 +19,14 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.0
+version: 0.2.1
```

```suggestion
version: 0.2.0
```
This is the chart version, which is separate to the image.tag/appVersion